src/main/resources/db/migration/V32__iat_ddl.sql

```diff
@@ -0,0 +1,2 @@
+-- Remove the unique constraint on tft resource as it is incorrect
+ALTER TABLE tft_resource DROP CONSTRAINT IF EXISTS tft_resource_resource_id_inserted_date_key;
```

Comment:
This unique constraint was added when the tft tables were going to be more like the item table.  This is no longer needed based on the updated design.